i18n/miraheze/en.json

```
@@ -23,6 +23,7 @@
 	"miraheze-sidebar-donate": "Donate",
 	"miraheze-sitenotice-closed": "This wiki has been automatically [[m:Special:MyLanguage/Dormancy Policy|closed]] because there have been '''no edits''' or '''log actions''' made within the last 60 days. If you are a user (who is not the bureaucrat) that wishes for this wiki to be reopened, please request that at [[m:Requests for reopening wikis|Requests for reopening wikis]]. If this wiki is not reopened within 6 months it may be deleted. Note: If you are a bureaucrat on this wiki, you can go to [[Special:ManageWiki]] and uncheck the \"Closed\" box to reopen it.",
 	"miraheze-sitenotice-closed-private": "This wiki has been automatically closed because there have been '''no edits''' or '''log actions''' made within the last 60 days. Since this wiki is private, it cannot be reopened by any user through the normal reopening request process. If this wiki is not reopened within 6 months, it may be deleted. Note: If you are a bureaucrat on this wiki, you can go to [[Special:ManageWiki]] and uncheck the \"Closed\" box to reopen it.",
+	"miraheze-sitenotice-closed-locked": "This wiki has been closed because Stewards or Trust and Safety have determined that it violates Miraheze's [[m:Content Policy|Content Policy]] or other global policies. For more information about why this wiki was closed, please see [[m:Special:Log/managewiki|the ManageWiki log]] on Meta or [[m:Help center|contact Stewards]] for more information. If you are a bureaucrat on this wiki and wish to appeal the closure you can do so [[m:Stewards' noticeboard|here]].",
```

First "for more information" probably not needed?

```suggestion
	"miraheze-sitenotice-closed-locked": "This wiki has been closed because Stewards or Trust and Safety have determined that it violates Miraheze's [[m:Content Policy|Content Policy]] or other global policies. Please see [[m:Special:Log/managewiki|the ManageWiki log]] on Meta or [[m:Help center|contact Stewards]] for more information. If you are a bureaucrat on this wiki and wish to appeal the closure you can do so [[m:Stewards' noticeboard|here]].",
```